components/slider.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Icon link -->
+    <link rel="icon" href="../assets/icon.png">
+    <!-- css style links -->
+
+    <link rel="stylesheet" href="../all_component.css">
+
+    <!-- title -->
+    <title>Slider</title>
+
+    <!-- Font Awesome link -->
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"
+        integrity="sha384-Bfad6CLCknfcloXFOyFnlgtENryhrpZCe29RTifKEixXQZ38WheV+i/6YWSzkz3V" crossorigin="anonymous" />
+
+</head>
+
+<body>
+    <!-- navbar-section -->
+    <header>
+        <nav class="navigation">
+            <div class="nav-brand"><a class="link" href="/index.html">Rebalance-UI</a></div>
+            <ul class="non-bullet">
+                <li class="list-tem-inline"><a class="link link-active" href="/index.html">Home</a></li>
+                <li class="list-tem-inline"><a class="link" href="./avatar.html">Docs</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <!-- side-list bar -->
```

Comment:
commented code makes it easier to scan, nice!

Reply:
Thank you @vinc3nati